services/app-web/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -70,7 +71,10 @@ const StatusTag = ({
 export const Dashboard = (): React.ReactElement => {
     const { loginStatus, loggedInUser } = useAuth()
     const location = useLocation()
-    const { testFrontendBanner } = useFlags()
+
```

Comment:
Curious what happened with `useFlags` and why that didn't work?  That looks more similar to what I have used before

Reply:
I was having some issues with registering users into LD and moved over to using `ldClient.variation`, which is what their docs had been referencing when talking about user segmentation. I can test out `useFlags` again though and see if we can get the same behavior again!